### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by wpreimes**
Combine buttons in results list with a dropdown list

### CODE REVIEW

**Comment by sheenaze** [comment]
It is nicely done :)
Like I said, I don't like this More... but I think I mainly don't like the color of it, maybe gray or generally something lighter :)
Also, if I have no plots generated the button for downloading plots is still there.. It is not something you spoiled, it had been like that before, but maybe you would like to fix it :)

**Comment by wpreimes** [comment]
yeah, maybe it should look more like a button, at the moment it is just text. Also if we already have a dropdown menu, the help text could be included

**Comment by sheenaze** [comment]
Help text?

**Comment by wpreimes** [comment]
> 
> 
> Help text?

instead of just showing the button it could also say what the button does in the menu

**Comment by sheenaze** [comment]
Ok :)

File: validator/migrations/0028_auto_20210216_2040.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        ('validator', '0027_auto_20210115_1024'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='validationrun',
+            name='scaling_method',
+            field=models.CharField(choices=[('none', 'No scaling'), ('min_max', 'Min/Max'), ('linreg', 'Linear regression'), ('mean_std', 'Mean/standard deviation'), ('cdf_beta_match', 'CDF matching with beta distribution fitting')], default='mean_std', max_length=20),
+        ),
+    ]
```
Comment (s-scherrer) [inline validator/migrations/0028_auto_20210216_2040.py:18]: Is this in this PR on purpose? Seems a bit unrelated.

Comment (wpreimes) [reply validator/migrations/0028_auto_20210216_2040.py:18]: no, looks like something that came from the update scaling

Comment (s-scherrer) [reply validator/migrations/0028_auto_20210216_2040.py:18]: Ok, it looks like this is not yet in the master branch, so maybe you can just leave it in here.

**Comment by sheenaze** [comment]
Nicely done :) two questions:
1. can the space between icons and ">>" button be smaller?
2. maybe ">>" should have the same color as buttons, so it suggests that there are more options?

**Comment by wpreimes** [comment]
> 
> 
> Nicely done :) two questions:
> 
>     1. can the space between icons and ">>" button be smaller?
> 
>     2. maybe ">>" should have the same color as buttons, so it suggests that there are more options?

I think the main question are
- Should we add text and/or make a "real" button?
- Should the menu open on click or mouse over?

I can make the space smaller and make the arrow blue

**Comment by sheenaze** [comment]
I kind of like those arrows :) and I think hovering should activate it.

**Comment by wpreimes** [comment]
image

**Comment by wpreimes** [comment]
image

**Comment by sheenaze** [comment]
Very nice :)

**Review by sheenaze — APPROVED** [review]
